src/components/ListingDetails/index.tsx

```diff
@@ -91,7 +99,12 @@ const caseInterpretationFields: ListingField<CaseListing>[] = [
   // Remote/In Person
```

Comment:
could we update the `parseTimeCommitment` function to also display "Not Available" instead of "N/A"?